lib/WUI/wui_api.h

```diff
@@ -49,6 +49,9 @@ typedef struct {
     char printer_state[PRI_STATE_STR_LEN]; // state of the printer, have to be set in wui
 } printer_info_t;
 
+typedef char system_time_t[12]; // wrapped string to ensure useage of at least 12 chars
+typedef char system_date_t[13]; // wrapped string to ensure useage of at least 13 chars
```

Comment:
This comment is:
1. Redundant. It is clear from code, that it is array of chars with size 12 (or 13). No need to state it here. There is no point of documenting something obvious like:
'''
c = a + b // This expression sums variable a and variable b and stores result in variable c.
'''
2. Misleading. At this point nothing is wrapped. This is definition of ordinary array of characters.
3. On the other hand what is stored inside should be documented in function documentation.

Reply:
commit `e98f5b1`